common/components/utils/ProjectAPIUtils.js

```diff
@@ -72,10 +74,22 @@ class ProjectAPIUtils {
       name: apiData.project_name,
       thumbnail: apiData.project_thumbnail,
       claimed: apiData.project_claimed,
-      date_modified: apiData.project_date_modified
+      date_modified: apiData.project_date_modified,
+      url: apiData.project_url,
+      positions: apiData.project_positions && apiData.project_positions.length !=0
```

Comment:
The null/empty check can be covered with _.isEmpty()

Reply:
Do you want me to apply this to the other checks in the same function, issue area and project stage?

Reply:
If it's code you wrote as part of this change, sure.  Otherwise, I'd say let's not expand the scope of this large pr.

Reply:
It's not, so I won't expand scope.